### ISSUE TITLE
Questions about reproducing the Llama3.2-1B 16 bit baseline result in the paper

### ISSUE THREAD

**Issue by FlyingPotatoZ** [facebookresearch/SpinQuant#36]
There is such data in Table 1 of the paper, but I cannot reproduce the data in this repository:

-  LLaMA-3.2 1B, 16-16-16, Wiki PPL=13.4

I use `train()` in `ptq.py`, call `data_utils.get_wikitext2` and `eval_utils.evaluator()`, set `bsz=1`, and used the LLaMA-3.2 1B original model as input to get the following results

- FP16 format: PPL=9.7483
- BF16 format: PPL=9.7548

Then I did the same experiment using the Llama-3.2-1B-Instruct model and reproduced the results similar to the paper:
- FP16 format: PPL=13.1538
- BF16 format: PPL=13.1585

I am confused by the description in the paper as it seems to describe two different models:

- The following content mentioned in Section _4 EXPERIMENTS_ of the paper:

> "We conduct experiments on the LLaMA-2 (Touvron et al., 2023b) models (7B/13B/70B), **LLaMA-3 (AI@Meta, 2024) models (1B/3B/8B)** and Mistral (Jiang et al., 2023) 7B model."

- and the following content mentioned in _A.10 FEW-SHOT RESULTS ON INSTRUCTION-FINETUNED MODELS_ of the paper:

> We further conduct experiments applying SpinQuant to **instruction-finetuned LLaMA 3.2 1B** and 3B models in Table 18. We present the results for few-shot learning scenarios. SpinQuant W4A8 quantized models demonstrate significant improvements in 5-shot accuracy on the MMLU benchmark and 1-shot rouge score on the TLDR9 summarization benchmark. It significantly closed the gap to the BF16 baseline.

"LLaMA-3 (AI@Meta, 2024) models (1B/3B/8B)" and "instruction-finetuned LLaMA 3.2 1B" both refer to Llama-3.2-1B-Instruct model, right? 

In addition, I would like to confirm whether the 16-bit baseline of Llama3.2-1B is in the BF16 data format.
